.github/workflows/pr-preview.yml

```diff
@@ -0,0 +1,37 @@
+name: PR Preview
+on:
+  pull_request: {}
+
+jobs:
+  dispatch:
+    runs-on: ubuntu-latest
+    permissions:
+        pull-requests: write
+    env:
+      branch_name: ${{ github.event.pull_request.head.ref }}
+      GH_TOKEN: ${{ github.token }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '16'
+          cache: npm
+      - uses: actions/cache@v3
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - run: git config --global url."https://github.com/".insteadOf ssh://git@github.com/
+      - name: Install dependencies
+        run: npm ci
+      - name: Configure git
+        run: |
+          git config --global url."https://${{ secrets.WAI_GIT_NAME }}:${{ secrets.WAI_GIT_ACCESS_TOKEN }}@github.com".insteadOf "https://github.com"
+          git config --global user.name "${{ secrets.WAI_GIT_NAME }}"
+          git config --global user.email "${{ secrets.WAI_GIT_EMAIL }}"
+      - name: Build and deploy WAI update
+        run: npm run pr:preview
```

Comment:
This seems to miss setting up the `BRANCH_NAME` environment variable that the script uses.

Reply:
Isn't the env block (lines 10 -- 11) a possible ways of doing this?

Reply:
It is. My bad. I read that file twice looking for an env setup and missed that 🙈